assignments/callbacks.js

```diff
@@ -24,25 +24,55 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
```

Comment:
good job with the callbacks. 

You've demonstrated you understand, and can execute on, the concept taught. Very good.